### PR TITLE
Logos

### PR DESCRIPTION
**Pull request by MichaelCGrant**
fixed logos

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,41 +1,20 @@
...
   "devtools_page": "./devTools.html",
 
-  "action": {
-    "default_title": "Sveltool"
-  },
-
-  "background": {
-    "service_worker": "./background.js"
+  "icons": {
+    "16": "./icons/icon_16.png",
```
Comment (Meowmerry) [inline public/manifest.json:10]: Nice logo, I love it.

File: src/utils/d3TreeRender.js [hunk 2]
```diff
@@ -6,9 +6,7 @@ import { componentProps, componentState, currentComponent } from './store';
   'use strict';
 })();
 
-let tree = d3.tree;
-let hierarchy = d3.hierarchy;
-let select = d3.select;
+let { tree, hierarchy, select } = d3;
```
Comment (Meowmerry) [inline src/utils/d3TreeRender.js:9]: Good destructuring!

**Review by Meowmerry — APPROVED** [review]

**Review by Meowmerry — APPROVED** [review]

File: src/utils/d3TreeRender.js (hunk 2 above)
Comment (MichaelCGrant) [reply src/utils/d3TreeRender.js:9]: Thanks!

File: public/manifest.json (hunk 1 above)
Comment (MichaelCGrant) [reply public/manifest.json:10]: Thanks!

**Pull request merged**